beacon-chain/blockchain/service.go

```diff
@@ -335,10 +335,6 @@ func (c *ChainService) receiveBlock(block *pb.BeaconBlock) error {
 		return fmt.Errorf("could not execute state transition %v", err)
 	}
 
-	if state.IsValidatorSetChange(beaconState, block.Slot) {
```

Comment:
deprecated, we moved it to transition.go, validator set change is now part of epoch processing